### ISSUE TITLE
Implement command builtin

### ISSUE THREAD

**Issue by timetoplatypus** [oilshell/oil#50]
Running `command -v clear`, for example, yields the following error message in OSH:
```
Unexpected error in execvpe('command', ['command', '-v', 'clear'], ...): [Errno 2] No such file or directory
```

**Comment by andychu** [comment]
OK so you use this command interactively?  I tend to use `which clear` instead, but we should have the bash version too.

**Comment by timetoplatypus** [comment]
`which` is from the GNU project I believe, so it may not always be on the system. `command` is more generally a UNIX/POSIX builtin convention, and I think if OSH wants to be a UNIX shell it should include this.

**Comment by andychu** [comment]
Yeah I was saying we should have it.  But I didn't know it was in POSIX:

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html

And this thread says `which` is not in POSIX: https://stackoverflow.com/questions/762631/find-out-if-a-command-exists-on-posix-system

**Comment by timetoplatypus** [comment]
Yup, `command` is pretty useful and it's common to see in shell scripts in general (not just bash scripts)

**Comment by andychu** [comment]
OK, this is done for `-v` now.  Let me know if you run into problems.

**Comment by timetoplatypus** [comment]
Just updated my OSH install to include the latest commits. I can verify that the `command` builtin is functioning with the -v flag.

**Comment by andychu** [comment]
Fix released with 0.3.
